### ISSUE TITLE
[Teamserver-Client--Bug]: Make on client error.

### ISSUE THREAD

**Issue by GitHubUserPy** [HavocFramework/Havoc#247]
### What happened?

When I run make on Arch Linux I have an error (_obviously, otherwise I would not be making this_). 
I do not think I need to give a screenshot, because what is going on is probably pretty clear.
`CMake Error at /usr/bin/cmake/Qt5/Qt5Config.cmake:28` 
I think it is because I have not install Qt5 on Arch Linux. Or maybe I have? I have run this on Arch Linux:
`sudo pacman -S qt5-base qt5-webengine qt5-tools` but I still get the error. What am I supposed to do?
If you want a screenshot, I can send you one, 
Thanks

### Did You Do a Pull First?

Latest (You performed a pull first)

### Did You Read Over Your Issue First?

- [X] I declare I made an effort and provided the necessary information for replication of the issue.

**Comment by winterrdog** [comment]
I don't think this an error caused by Havoc( based on the directory path of the erratic file )

It's an issue with some of the software versions installed on your system 

The issue might be with QT5

**Comment by GitHubUserPy** [comment]
Alright! I'll try to figure it out myself! Thanks
